package.json

```diff
@@ -14,9 +14,9 @@
     "test-e2e": "NODE_PATH=. node ./node_modules/codeceptjs/bin/codecept.js run -c ./src/test/end-to-end/ --steps",
     "test-crossbrowser-e2e": "NODE_PATH=. codeceptjs run-multiple ${BROWSER_GROUP:-'--all'} -c src/test/end-to-end/saucelabs.conf.js --plugins retryFailedStep --steps --grep '@crossbrowser' --reporter mocha-multi",
     "test:functional": "NODE_PATH=. node ./node_modules/codeceptjs/bin/codecept.js run-multiple parallel -c ./src/test/end-to-end/ --steps --reporter mochawesome",
-    "test:fullfunctional-pr": "./bin/run-e2e-tests-pr.sh",
-    "test:fullfunctional": "./bin/run-e2e-tests.sh",
-    "test:fullfunctional-cw": "./bin/run-e2e-tests-cw.sh",
+    "test:fullfunctional-pr": "E2E_TEST_PATH='./paths/**/pr_specific/*.js' ./bin/run-e2e-tests.sh",
+    "test:fullfunctional": "E2E_TEST_PATH='./paths/**/*.js' ./bin/run-e2e-tests.sh",
+    "test:fullfunctional-cw": "E2E_TEST_PATH='./paths/caseworker/*.js' ./bin/run-e2e-tests-cw.sh",
```

Comment:
the -cw.sh is still referenced here, but you deleted it?

Reply:
missed that, resolved.